src/main/java/mate/controller/GetAllDriverByCarIdController.java

```diff
@@ -0,0 +1,30 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/driversByCar")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/drivers")
```

look to the [checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice) there is some suggestions about url naming